framework/doc/content/source/interfaces/SolutionInvalidInterface.md

```diff
@@ -1,7 +1,19 @@
 # SolutionInvalidInterface
 
-The SolutionInvalidInterface defines the method used to mark a solution as "invalid".  An invalid solution means that the solution somehow does not satisfy requirements such as a value being out of bounds of a correlation.  Solutions are allowed to be invalid _during_ the nonlinear solve - but are not allowed to invalid once it converges.  A "converged" solution that is marked as invalid will cause MOOSE to behave as if the solution did NOT converge - including cutting back timesteps, etc.
+## Tracking warning occurences
+
+The `SolutionInvalidInterface` tracks the occurences of `mooseWarning` and `paramWarning` across all [MooseObject.md] and [Action.md].
```

Comment:
Does this mean there is no `mooseWarning` or `paramWarning` during run time? If so, that should be mentioned.

Reply:
Got it. It would be nice if we can have an option to silence them during the simulation since all are reported in the end. But it doesn't need to be this PR. Good with me then